tuplex/core/src/physical/ResolveTask.cc

```diff
@@ -228,7 +228,7 @@ namespace tuplex {
 
         // needs to be put into separate list of python objects...
         // save index as well to merge back in order.
-        _py_nonconfirming.push_back(std::make_tuple(_rowNumber, out_row));
+        _py_nonconfirming.push_back(std::make_tuple(_rowNumber - _numUnresolved, out_row));
```

Comment:
is _rowNumber >= _numUnresolved always true? Else, this could lead to a bad overflow issue...
maybe add an assert that this operation is safe.